frontend/Dockerfile.debug

```diff
@@ -0,0 +1,31 @@
+FROM node:16.16-alpine as build
+
+WORKDIR /app
+ENV PATH /app/node_modules/.bin:$PATH
+
+# frontend-sdk
+COPY frontend-sdk/package.json frontend-sdk/
+COPY frontend-sdk/package-lock.json frontend-sdk/
+
+WORKDIR /app/frontend-sdk
+RUN npm ci --silent
+COPY frontend-sdk/ ./
+RUN npm run build
+
+
+# elements
+WORKDIR /app/elements
+COPY elements/package.json ./
+COPY elements/package-lock.json ./
+
+RUN npm ci
+COPY ./elements ./
+RUN npm run build:dev
```

Comment:
Would be nice if the steps here would also align, like the one in the normal `Dockerfile`, except for the `build:dev` part.

Reply:
👍